dev/com.ibm.ws.jaxws.2.3.web/resources/com/ibm/ws/jaxws/web/internal/resources/JaxWsWebMessages.nlsprops

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+# NLS_MESSAGEFORMAT_ALL
+#
+#CMVCPATHNAME com.ibm.ws.jaxws.web/resources/com/ibm/ws/jaxws23/internal/resources/JaxWsWebMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#COMPONENTPREFIX CWWKW
+#COMPONENTNAMEFOR CWWKW WebSphere JAXWS 2.3 Web Code
+#RANGE 0400 - 0499
+#
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+#{0} is the servlet name
+error.urlPattern=CWWKW0400E: The url-pattern of the servlet-mapping for servlet {0} must not contain an asterisk (*).
+error.urlPattern.explanation=The url-pattern of the servlet-mapping must be an exact match pattern and must not contain an asterisk.
+error.urlPattern.useraction=Make sure that url-pattern of servlet-mapping of the servlet is an exact match pattern.
+
```

Comment:
Looks like it no changes are needed.

Reply:
thanks!

Reply:
FYI, in a future PR we are considering reusing the JAXWS messages, so these MAY get pulled out.